tools/flaky-test-retryer/github_commenter.go

```diff
@@ -59,7 +59,7 @@ func NewGithubClient(githubAccount string, dryrun bool) (*GithubClient, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &GithubClient{ghc, *user.Login, dryrun}, nil
+	return &GithubClient{ghc, user.GetID(), dryrun}, nil
```

Comment:
GetID return int64 instead of ptr, right?

Reply:
Yes, seems the getter funcs strip the pointer off for us: https://godoc.org/github.com/google/go-github/github#User.GetID